tests/testthat/test_XcmsExperiment.R

```diff
@@ -1326,3 +1326,34 @@ test_that("storeResults,RDataParam works", {
     expect_s4_class(object, "XcmsExperiment")
     expect_equal(object, xmse)
 })
+
+test_that("storeResults,PlainTextParam,MsExperiment works", {
+    param <- PlainTextParam(path = "/path/test/")
+    param2 <- PlainTextParam()
+    expect_false(is.null(param2))
+    # Test if PlainTextParam is invalid with multiple paths
+    expect_error(new("PlainTextParam", path = c(tempdir(), tempdir())))
+    mse <- filterRt(mse, c(100,150)) #this adds to processing but not processingQueue
```

Comment:
filtering on retention time might actually already do the filtering and not put that into the processing queue (because it is the same as just subsetting the `Spectra`). Operations performed on peak data will however be put into the processing queue, so better to call e.g. `mse <- filterMzRange(mse, c(200, 500))`.